common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -845,3 +845,202 @@ def test_generate_certificate_exception(self):
                              ' below to send the certificate.',
             self.certificates_section.message.text
         )
+
+
+@attr('shard_1')
+class CertificateInvalidationTest(BaseInstructorDashboardTest):
+    """
+    Tests for Certificates functionality on instructor dashboard.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        super(CertificateInvalidationTest, cls).setUpClass()
+
+        # Create course fixture once each test run
+        CourseFixture(
+            org='test_org',
+            number='335535897951379478207964576572017930000',
```

Comment:
Is it possible to split out this number info a parameter/field which can be referenced, so we do not create a fragile situation where we have the same value repeated in multiple locations?  (DRY and all...)